lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -91,6 +91,14 @@ def _on_install(self, _) -> None:
 _kebabize = re.compile(r"(?<=[a-z0-9])(?=[A-Z])")
 
 
+class SlurmOpsError(BaseException):
```

Comment:
Why `BaseException` instead of `Exception`. [This documentation piece from Python](https://docs.python.org/3/library/exceptions.html) seems to indicate that we should derive our custom exception from `Exception` instead. 

Tbh we could copy the exception from `is_container` which just returns the first arg passed to the exception as the message.